### PR TITLE
Parent style dev complete

### PR DESCRIPTION
**Pull request by simrin051**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+}
+
+a .logo {
+    font-size: 25px;
+}
+
+nav {
+    width: 100%;
+    background-color: var(--secondary-color) !important;
```
Comment (srishtigautam21) [inline style.css]: As vikram suggested do not use !important

File: style.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
+:root {
+    --primary-color: #9400d3;
+    --secondary-color: #CBC3E3;
+}
+
+.title {
+    font-size: 24px;
+}
```
Comment (srishtigautam21) [inline style.css:9]: You should use rem for anything thats greater than 10px or 16px

**Review by srishtigautam21 — APPROVED** [review]
Good to merge with suggested changes

File: style.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+.margin-btm-5 {
+    margin-bottom: 5px;
+}
+
+.margin-btm-10 {
+    margin-bottom: 10px;
+}
+
+.title-underline {
+    border-bottom: 5px solid red;
```
Comment (sri-rishi) [inline style.css]: Suggestion: use variable color naming for colors

File: style.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 15px;
+    text-decoration: none;
+}
+
+body {
+    margin: 0px;
+}
```
Comment (sri-rishi) [inline style.css:98]: Suggestion: using `0 ` here will work

**Review by sri-rishi — COMMENTED** [review]
Good to merge, code looks clean

**Pull request merged**